Formula/libxlsxwriter.rb

```diff
@@ -0,0 +1,35 @@
+class Libxlsxwriter < Formula
+  desc "Libxlsxwriter: A C library for creating Excel XLSX files."
+  homepage "https://libxlsxwriter.github.io/"
+  url "https://github.com/jmcnamara/libxlsxwriter/archive/RELEASE_0.7.3.tar.gz"
+  sha256 "84dbbd164b440695119ded3a8e2b6cac07d0d284343ab977c88dc0418d8fcde5"
+
+  depends_on "lzlib"
```

Comment:
[Web page](https://github.com/jmcnamara/libxlsxwriter) says zlib is the only dependency. Why `lzlib` here?